draft-ietf-httpbis-resumable-upload.md

```diff
@@ -89,17 +89,19 @@ The terms Byte Sequence, Item, String, Token, Integer, and Boolean are imported
 
 The terms "representation", "representation data", "representation metadata", "content", "client" and "server" are from {{HTTP}}.
 
+The term "patch document" is taken from {{RFC5789}}.
```

Comment:
While not strictly related to the topic of the PR, I noticed that we were missing a reference to where _patch document_ comes from.